src/blocks.js

```diff
@@ -32,6 +48,48 @@ export default (editor, opts = {}) => {
     });
   });
 
+  /*
+   * Blocks for Preference Center Category
+   */
+  //check if page is for preference center
+  if (mQuery('#page_isPreferenceCenter_1').is(':checked')) {
```

Comment:
to make the code better readable and more robust this could be converted to a function. E.g.: isPreferenceCenter()

Reply:
Hi @adiux , its done

Reply:
there is another occurence of this check on line 56